packages/pigeon/tool/test.dart

```diff
@@ -18,11 +18,13 @@ import 'shared/test_runner.dart';
 import 'shared/test_suites.dart';
 
 const String _testFlag = 'test';
+const String _genFlag = 'skip-generation';
 const String _listFlag = 'list';
 
 Future<void> main(List<String> args) async {
   final ArgParser parser = ArgParser()
     ..addMultiOption(_testFlag, abbr: 't', help: 'Only run specified tests.')
+    ..addFlag(_genFlag, abbr: 'g', help: 'Skips the generation step.')
```

Comment:
You probably want to make this non-negatable (or referring or whatever it's called, where by default you can do `-no-whateverflag`).